### ISSUE TITLE
Maya: settings for multiple review instances

### ISSUE THREAD

**Issue by m-u-r-p-h-y** [ynput/OpenPype#3273]
as we have a possibility of having multiple review instances now #3269

we need to come up with a system of using different settings for each instance.

The current practice of picking up and exposing arbitrary attributes (like Transparency in this example) 
![image](https://user-images.githubusercontent.com/352795/171439216-e3b6bc00-5d5f-4dc9-b63f-03270f1dd4f2.png)
is very bad, cluttering the code and UI

examples of different settings:

- I can have two different static images/turnarounds for modeling review - one shaded, the other with wireframe for topology approval
- animation playblasts - one with a simple image plane, the other with alpha for comp to add more sophisticated one
- etc.

[cuID:[OP-3357](https://app.clickup.com/t/2eqzt5n)]

**Comment by mkolar** [comment]
Yes if we convert maya to the new publisher this would be solve pretty much. We'd still need to pick arbitrary attributes that would be exposed in there, but I think that's more question of giving ability to store multiple review presets that could be applied to various reviews.
